components/sidebar.js

```diff
@@ -1,88 +1,97 @@
-import React from 'react'
-import DonutLargeIcon from '@material-ui/icons/DonutLarge';
-import ClearAllIcon from '@material-ui/icons/ClearAll';
-import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward';
-import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
-import SyncAltIcon from '@material-ui/icons/SyncAlt';
-import LayersIcon from '@material-ui/icons/Layers';
-import LockIcon from '@material-ui/icons/Lock';
-import EcoIcon from '@material-ui/icons/Eco';
+import React from "react";
+import { makeStyles } from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import Paper from "@material-ui/core/Paper";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import Card from "./Cards/Card";
+import AccountCircleIcon from "@material-ui/icons/AccountCircle";
+import MenuOpenIcon from "@material-ui/icons/MenuOpen";
+import HomeIcon from "@material-ui/icons/Home";
+import SearchIcon from "@material-ui/icons/Search";
+import EqualizerIcon from "@material-ui/icons/Equalizer";
+import LayersIcon from "@material-ui/icons/Layers";
+import FriendCard from "./Cards/FriendCard";
 
-const Sidebar = () => {
-    return (
-        <div className="md:w-3/12 w-6/12 h-screen shadow-2xl">
-            <div className=" border-b py-3 mt-1 flex justify-around ">
-                <p className="text-xl  font-semibold">elrond</p>
-                <p>|</p>
-                <p className="text-gray-400 text-lg">wallet</p>
+const useStyles = makeStyles({
+  button: {
+    fontSize: "18px",
+    textTransform: "none", //makes it lowercase
+    maxWidth: "200px",
+    maxHeight: "50px",
+    minWidth: "200px",
+    minHeight: "30px",
+    backgroundColor: "transparent",
+    color: "#BDACD4",
+    "&:hover": {
+      backgroundColor: "#977BBF",
+      color: "#fff",
+    },
+  },
+  box: {
+    borderRadius: "0px 15px 15px 0px",
+    // HOW TO PUT MATERIAL UI COLORS HERE
+    background: "linear-gradient(145deg, #4E3276, #8569AE)",
+    height: "100%",
+  },
+});
```

Comment:
not sure if you did this or if this was in the template. Please avoid makeStyles! =D